deployment/frozen/mainnet.toml

```diff
@@ -1 +1,4 @@
+# version v0.2.2
 [[code_hash_list]]
+cluster = "7366a61534fa7c7e6225ecc0d828ea3b5366adec2b58206f2ee84995fe030075"
+spore = "9cb02fa8621d79fff300b910d52665f594d3ad468142a66213cfaaca0409502c"
```

Comment:
```bash
# checkout commit b562411fe3082d1362bae7a0d15c32ff94f80e0f tag: 0.2.2
capsule clean
capsule build --release -- --features release_export
```

And then I check the code_hashes:

## commit b562411fe3082d1362bae7a0d15c32ff94f80e0f (capsule build --release -- --features release_export)
- code_hash of contact spore              : 0x98b14c35863d82973b45fdc44c82dc04570dcde11c97f5c41f2b71c37dfa905f
- code_hash of contact cluster            : 0x7366a61534fa7c7e6225ecc0d828ea3b5366adec2b58206f2ee84995fe030075

### Could you check the code_hash of contact spore again?

Reply:
I know it, it caused by the new appending `frozen` version, so the Spore binary changed by supporting compatible `code_hash`

Reply:
@Flouse I have to change the linked commit hash of tag 0.2.2, so please change it again in your local machine

Reply:
Could you tell me exactly which commit you used to build and deploy to https://github.com/sporeprotocol/spore-contract/blob/f4d8f90a52fae3267442b3beaf14fda5c633b279/deployment/migration/mainnet/spore/2024-01-23-40ef57a2.json

Reply:
commit hash is not correct, I have to find out later

Reply:
> @Flouse I have to change the linked commit hash of tag 0.2.2, so please change it again in your local machine

Again, I suggest avoid changing tag, see https://github.com/sporeprotocol/spore-contract/pull/52#discussion_r1462794837 and https://git-scm.com/docs/git-tag#_on_re_tagging

You can simply create new tags such as 0.2.3, 0.2.4, if you need.